docs/content/drafts/Button2.mdx

```diff
@@ -65,6 +65,20 @@ The `invisible` variant of `Button` indicates that the action is a low priority
 </>
 ```
 
+### Selected button
+
+`Button` can be set to in a "selected" state in two ways. You can use a `selected` class or set `aria-selected ="true"`.
+This is in many cases similar to the active state of the button. Usually used in `ButtonGroup` when like tabs.
```

Comment:
Should this capability be restricted to `ButtonGroup`?  I can see two risks with enabling this for individual buttons:

1) Opens door to misuse of standalone button appearance
2) A standalone button shouldn't need to be pre-selected

Reply:
🤔 I don't understand what misuse of this pattern could look like.
How would you restrict to `ButtonGroup`? By parent class?

Reply:
We talked about this in the Primer React planning meeting.

I don't think we want to restrict it to `ButtonGroup` buttons, especially because we use the "selected" styling when a button controls a dropdown and the dropdown is open.

https://user-images.githubusercontent.com/2313998/151037767-e41232dc-e1db-46cd-9724-4814d5575a8f.mp4

Reply:
I _think_ what we decided in the meeting is to utilize the `active` state for dropdowns (this is all forward thinking, not talking about existing functionality) and restrict `aria-pressed` to a future **new** Segmented Control component (a more specific version of a ButtonGroup). The confusing piece of this is that we are simultaneously updating an existing Button component while refactoring a future Button component.

@pksjce I may have missed this, but do you have the use case that prompted this ask to add `selected`? We may need to continue supporting it in the original Button while we redesign the future Button.. but my recommendation there will depend on the use case for adding it now.

Reply:
@langermank - this was the original issue where this came up https://github.com/primer/react/issues/1783

Reply:
Thanks @rezrah.. that's an interesting use case 🤔 I still think this would be a `SegmentedControl` though I imagined that to be two/three options max not six 😆 I guess in that case to unblock them use the `aria-selected` styles from Primer CSS but map it to `aria-pressed`? But I would only add this to the existing Button component, not the refactor.

Reply:
We are planning to deprecate the existing component as soon as possible. I think for parity we'd need to have it in the new one as well? The new component is not a refactor in terms of functionality I feel, only the API. Once you are done with https://github.com/primer/css/pull/1874, we'd have to work on absorbing it into primer-react.

Reply:
@pksjce right, thanks for pointing that out. How do you see the new Button API changes rolling into PRC? In CSS and PVC, I think we'll likely end up creating a new component similar to how you've handled `Button2`. If you follow that pattern in PRC, you would end up with `Button3` 😆 Although, the markup in `Button2` is almost identical to where we're going with the Button API changes, so perhaps it wouldn't be a big deal to wrap it into `Button2` once `Button` is deprecated (this is getting a little confusing!)

Reply:
Yes, I'm not planning to have a `Button3` for sure. I'd just update `Button2` which would become the only `Button` and release it going forward. I don't think there'd be any major API changes. If any breaking then we'd go for a major version change I believe.